src/components/NewMovie/NewMovie.tsx

```diff
@@ -69,7 +73,7 @@ export const NewMovie: React.FC<Props> = ({ onAdd }) => {
         label="Image URL"
         value={imgUrl}
         onChange={setImgUrl}
-        validation={(isValidUrl)}
+        validate={(checkIsValidUrl)}
```

Comment:
```suggestion
        validate={checkIsValidUrl}
```

Unnecessary brackets